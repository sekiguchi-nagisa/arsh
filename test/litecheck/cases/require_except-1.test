
ls -la

# RUN: /bin/sh $self
# CHECK: 123
# REQUIRE: (34/0) == 0