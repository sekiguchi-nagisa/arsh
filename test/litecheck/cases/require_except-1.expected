invalid REQUIRE directive: `(34/0) == 0', caused by
zero division